src/widget/text-print/text-print.js

```diff
@@ -25,17 +26,23 @@ class TextPrintWidget extends Widget {
         this.element.after( printElement );
 
         this.widget = this.element.parentElement.querySelector( `.${className}` );
-        this._copyValue();
-
-        this.element.addEventListener( 'input', () => {
-            this._copyValue();
-        } );
+        this.element.classList.add( 'default-print' );
+        const comment = this.question.parentElement.querySelector( '[role="comment"]' );
+        $( comment )
```

Comment:
this is for showing text area/input value when user show print preview without API(e.g. like using ctrl+p) since the text area/input will be hidden on print preview and no value copied to text-print widget yet then without this class, the user will not see any value. This class is not needed previously since there is an input listener to automatically update the text-print widget.

Reply:
I was referring to 'comment'. :)

Reply:
ah my bad, previously I used comment/dn widget as the listener since the printify event dispatched from that widget.